modules/oas-page-builder/src/services/database.ts

```diff
@@ -26,6 +26,7 @@ const getDbName = () => {
   switch (env) {
     // Autobuilder's prd env
     case 'production':
+      return 'snooty_prod';
     case 'dotcomprd':
       return 'snooty_dotcomprd';
     // Autobuilder's pre-prd env
```

Comment:
If we're gonna add the mapping for the `production` environment, we should also add it for the `staging` environment for completeness